src/components/Register.vue

```diff
@@ -26,6 +26,15 @@
                             <i class="instagram large icon"></i>
                             Sign in with Instagram
                         </button>
+
```

Comment:
This isn't necessary.  As a dev, you should be able to login using any of the methods as a normal user and its only tiktok for now. The token is hardcoded in ln 51 of router/index.js file, for your local dev, update this token to a new one when needed. Kindly throw this token value in an dev environment variable. remove this block of code also remove the hardcoded email + password credentials for security purposes please don't commit sensitive data such as these.
You can always generate your tokens using postman, sent you the link couple days ago, I can resend it later